### PR TITLE
ef - Added get riders and get drivers plus tests

### PR DESCRIPTION
**Pull request by EKF626**
In this PR, we add the ability to get a list of all users designated as "rider" and all users designated as "driver".  To test, use swagger to get toggle the rider and driver status of some users then get riders and get drivers.

### CODE REVIEW

**Comment by TG6S85** [comment]
Looks like all checks are good to go

**Review by TG6S85 — COMMENTED** [review]
Looks good

**Review by TG6S85 — DISMISSED** [review]
MB

File: src/test/java/edu/ucsb/cs156/gauchoride/controllers/UsersControllerTests.java [hunk 1]
```diff
@@ -436,4 +437,66 @@ public void admin_tries_to_toggle_rider_status_non_existant_user_and_gets_right_
...
+    User u2 = User.builder().id(2L).rider(false).build();
+    User u3 = User.builder().id(3L).rider(true).build();
+    User u4 = User.builder().id(4L).rider(false).build();
+
+    ArrayList<User> expectedUsers = new ArrayList<>();
+    expectedUsers.addAll(Arrays.asList(u1, u3));
+    String expectedJson = mapper.writeValueAsString(expectedUsers);
+
+    ArrayList<User> allUsers = new ArrayList<>();
+    allUsers.addAll(Arrays.asList(u1, u2, u3, u4));
```
Comment (andrewpengucsb) [inline src/test/java/edu/ucsb/cs156/gauchoride/controllers/UsersControllerTests.java:456]: Adjust indentation to be consistent with rest of file

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

File: src/test/java/edu/ucsb/cs156/gauchoride/controllers/UsersControllerTests.java (hunk 1 above)
Comment (andrewpengucsb) [reply src/test/java/edu/ucsb/cs156/gauchoride/controllers/UsersControllerTests.java:456]: PR is mergable once fixed

**Review by andrewpengucsb — APPROVED** [review]
LGTM

**Review by cmfisher606 — APPROVED** [review]
Get functions and tests look good!

**Pull request merged**